packages/nc-gui/composables/useMapViewDataStore.ts

```diff
@@ -72,7 +72,10 @@ const [useProvideMapViewStore, useMapViewStore] = useInjectionState(
 
     async function loadMapMeta() {
       if (!viewMeta?.value?.id || !meta?.value?.columns) return
-      mapMetaData.value = await $api.dbView.mapRead(viewMeta.value.id)
+      mapMetaData.value = isPublic.value ? (sharedView.value?.view as MapType) : await $api.dbView.mapRead(viewMeta.value.id)
+
+      // if (!viewMeta?.value?.id || !meta?.value?.columns) return
+      // mapMetaData.value = await $api.dbView.mapRead(viewMeta.value.id)
```

Comment:
can be removed

Reply:
sorry - I missed to review my own PR :-/ :D